qute.ls/com.redhat.qute.ls/src/main/resources/com/redhat/qute/project/datamodel/qute-resolvers.jsonc

```diff
@@ -88,6 +88,24 @@
 			],
 			"url": "https://quarkus.io/guides/qute-reference#arrays"
 		},
+		// ------------- Character Escapes Resolvers --------------
+		// See https://quarkus.io/guides/qute-reference#character-escapes
+		{
+			"signature": "raw(base : java.lang.String) : io.quarkus.qute.RawString",

```

Comment:
It works with any Object ,so the signature should be

`"signature": "raw(base : java.lang.Object) : io.quarkus.qute.RawString"`